includes/blocks/class-sensei-quiz-blocks.php

```diff
@@ -34,6 +34,20 @@ public function enqueue_block_editor_assets() {
 		Sensei()->assets->enqueue( 'sensei-quiz-blocks', 'blocks/quiz/index.js', [], true );
 		Sensei()->assets->enqueue( 'sensei-quiz-blocks-editor', 'blocks/quiz/quiz.editor.css', [ 'sensei-shared-blocks-editor-style', 'sensei-editor-components-style' ] );
 
+		/**
+		 * Filters the quiz ordering type promo toggle.
+		 *
+		 * @hook  sensei_quiz_ordering_question_type_hide
+		 * @since 4.1.0
+		 *
+		 * @param  {bool} $hide_content_drip Whether to hide the ordering question type promo.
```

Comment:
Copy/paste error for the variable name?

Reply:
Yep :(

Reply:
Fixed here: https://github.com/Automattic/sensei/pull/4815/commits/4d25864103bd6ca715ec28c8c738e90ee50070a9